scipy/_lib/_docscrape.py

```diff
@@ -542,12 +542,12 @@ def __str__(self, func_role=''):
         return '\n'.join(out)
 
 
-def indent(str, indent=4):  # noqa
+def indent(str, indent=4):
```

Comment:
Or this one?

Reply:
using `str` as a variable name. If we ever wanted to enable the check for that, we would probably want this to be changed anyway.

Reply:
Yeah OK.